working/augmentation-libraries/feature-specification.md

```diff
@@ -923,20 +924,21 @@ present), and it may invoke the augmented body by calling
 
 #### Redirecting generative constructors
 
-A redirecting generative constructor marked `augment` adds its redirecting
-initializer to the augmented constructors initializer list.
+A redirecting generative constructor marked `augment` adds its redirection
+to the augmented constructor.
 
 This converts it into a redirecting generative constructor, removing the
 potentially non-redirecting property of the constructor.
 
 It is a compile-time error if:
 
-*   The augmented constructor has any initializers or a body.
+*   The augmented constructor has an initializer list or a body, or it has a
+    redirection.
```

Comment:
I assume this error was intended: "adds its redirection" sounds like the redirection position isn't currently occupied, otherwise we would say "adds or replaces its redirection". Also, the implied statement that the augmented constructor is potentially non-redirecting would not be true if it is already a redirecting generative constructor.

Reply:
I think @jakemac53 originally wrote this section but that logic sounds right to me.

Reply:
Sounds good, thanks!